docs/r3/beyondcandc.md

```diff
@@ -1,28 +1,29 @@
 # Beyond "command and control"
+
 "Command and control"-style interoperation, where system A instructs system B to do something, or makes decisions based on observable properties of system B, is largely a solved problem at the high level. There is always work to do in defining message standards and communications protocols, but the basic co-ordination mechanisms are well-understood:
```

Comment:
Is it worth casting this as an example to better introduce the reader to what you mean. I think there is a bit of a jump from the title of the page to the meat of what you're discussing that could be eased with an example of C&C